db/migrate/20150617184118_create_tasks.rb

```diff
@@ -1,11 +1,14 @@
 class CreateTasks < ActiveRecord::Migration
   def change
     create_table :tasks do |t|
-      t.string :name, null: false
-      t.string :description
+      t.string   :name, null: false
+      t.string   :description
       t.datetime :date_complete
+      t.integer  :person_id
```

Comment:
I probably still would create a new migration out of habit, but you don't have to, just wanted to bring up the idea.